package.json

```diff
@@ -34,6 +34,7 @@
     "@endo/eventual-send": "~0.16.5",
     "@endo/init": "0.5.49",
     "@headlessui/react": "^1.7.2",
+    "bech32": "^2.0.0",
```

Comment:
we've already got `@cosmjs/encoding`. instead of taking another dep can't we use https://cosmos.github.io/cosmjs/latest/encoding/modules.html#fromBech32 ?